docs/sqlite.md

```diff
@@ -0,0 +1,26 @@
+# SQLite support in LND
```

Comment:
Carrying over my comments from https://github.com/lightningnetwork/lnd/pull/7251 for convenience:
 - Mention that Windows i386/ARM and Linux PPC/MIPS aren't supported for SQLite backends (and why)
 - Mention that just switching the `db.backend` to another type will _NOT_ migrate any data, so for the time being the SQLite backend can only be used for _new_ nodes.